includes/class-wc-payments-order-service.php

```diff
@@ -687,7 +687,7 @@ public function get_wcpay_intent_currency_for_order( $order ) : string {
 	 *
 	 * @throws Order_Not_Found_Exception
 	 */
-	public function set_wcpay_refund_id_for_order( $order, $wcpay_refund_id ) {
+	public function set_wcpay_refund_id_for_refund( $order, $wcpay_refund_id ) {
```

Comment:
```suggestion
	public function set_wcpay_refund_id_for_refund( $wc_refund, $wcpay_refund_id ) {
```

Also update PHPdoc (just my suggestions but use your own judgemnt :D )

- Type `WC_Order_Refund` for the first param `$wc_refund`
- Description of method: Set WCPay refund ID as metedata for refund object.

Reply:
I admit I was a bit confused by the fact that we call `$this->get_order( $order )` in the body of this method. But then I understood the reason behind is to be able to use mocks for the sake of testing. I decided to change the references to `$order` in the body of the method to `$wc_refund`, hope it makes sense as well. You can check it in this commit https://github.com/Automattic/woocommerce-payments/pull/8200/commits/bc135bca5f8519a651d627bfe035cc613bc4de36 thanks!